src/main/flight/pid.h

```diff
@@ -299,6 +300,9 @@ typedef struct pidRuntime_s {
     pt1Filter_t ptermYawLowpass;
     bool antiGravityEnabled;
     pt2Filter_t antiGravityLpf;
+#ifdef USE_WING
+    pt2Filter_t tpaLpf;
+#endif
```

Comment:
New parameters should be at the end? (line 342)

Reply:
oh cmon lol :)
Moved